pom.xml

```
@@ -14,6 +14,18 @@
     <properties>
         <testcontainers.version>1.19.1</testcontainers.version>
     </properties>
+    <repositories>
+        <repository>
+            <id>infinispanSnapshots</id>
+            <releases>
+                <enabled>true</enabled>
```

AFAIK, `https://s01.oss.sonatype.org/content/repositories/snapshots` does not expose releases (Final, CRs, Dev...)
```suggestion
                <enabled>false</enabled>
```